internal/bminventory/inventory.go

```diff
@@ -4230,6 +4230,9 @@ func (b *bareMetalInventory) UpdateHostApprovedInternal(ctx context.Context, clu
 		log.WithError(err).Errorf("failed to update 'approved' in host: %s", hostId)
 		return err
 	}
+	hId := strfmt.UUID(hostId)
+	b.eventsHandler.AddEvent(ctx, strfmt.UUID(clusterId), &hId, models.EventSeverityInfo,
+		fmt.Sprintf("Updated Approved to %t Host %s Cluster %s", approved, hostId, clusterId), time.Now())
```

Comment:
```suggestion
		fmt.Sprintf("Host %s: updated approved to %t", hostutil.GetHostnameForMsg(&host), approved), time.Now())
```

(Need to get the host from the DB)

Reply:
Done